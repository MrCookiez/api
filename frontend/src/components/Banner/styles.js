import { css } from 'styled-components';
import theme from '../../config/theme';

export default css`
    display: flex;
    justify-content: center;
    align-items: center;
    width: 100%;
    padding: ${theme.spacing.sm}px 0;
    background-color: ${theme.colors.grayScale.light};
`;