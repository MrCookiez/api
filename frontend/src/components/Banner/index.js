import React from 'react';
import 'styled-components/macro';
import styles from './styles';

const Banner = () => (
    <div css={styles}>
        Banner
    </div>
);

export default Banner;
