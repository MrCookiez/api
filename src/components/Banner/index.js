import React from 'react';
import 'styled-components/macro';
import Logo from '../Logo';
import styles from './styles';

const Banner = () => (
  <div css={styles}>
    <Logo isWhite />
  </div>
);

export default Banner;
