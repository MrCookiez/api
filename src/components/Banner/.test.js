import React from 'react';
import { create } from 'react-test-renderer';
import Banner from '.';

describe('<Banner />', () => {
    it('should return match the snapshot', () => {
        const component = create(<Banner />);
        expect(component.toJSON()).toMatchSnapshot();
    });
});
