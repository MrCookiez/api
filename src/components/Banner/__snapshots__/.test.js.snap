// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Banner /> should return match the snapshot 1`] = `
<div
  className="Banner___StyledDiv-lpf000-0 eCHwFM"
>
  <div
    className="Logo___StyledDiv-v7wbvh-0 hslNiJ"
  >
    <svg
      width={25}
    >
      pepper.svg
    </svg>
    <a
      to="/"
    >
      florinafood.gr
    </a>
  </div>
</div>
`;
